docs/How-to-add-new-gem-to-omsagent-bundle.md

```diff
@@ -0,0 +1,49 @@
+## How to add a new gem to omsagent install bundle
+
+In order to add a new Ruby gem into OMSAgent bundle when you have the new_gem_name.gem file on you local system you need to follow these steps:
+1. Update the private branch:
+* Clone Microsoft/Build-OMS-Agent-for-Linux github repo
+* Copy the new_gem_name.gem to Build-OMS-Agent-for-Linux//omsagent/source\ext\gems folder.
+* Navigate to Build-OMS-Agent-for-Linux//omsagent/build and put modified commands in buildRuby.sh
+(Example ${RUBY_DESTDIR}/bin/gem install ${BASE_DIR}/source/ext/gems/new_gem_name.gem)
+* Run buildRuby.sh 100
```

Comment:
This assumes a lot, like that you're in the build directory, you ran configure, etc. You may want to just suggest that they configure and build OMS before following these instructions.